server/controller/middleware/validation.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+const getUser = require('../database/queries/getUser.js');
+
+exports.isFound = (req, res, next) => {
+  console.log(req);
+
+  getUser(req.body.email).then((resultIsFound) => {
+    console.log(resultIsFound);
+
+    if (!resultIsFound) {
+      console.log('not register');
+      res.end();
```

Comment:
@alaabadra This could return a 401.

Reply:
ok, thank you for this comment ,but this not final push only i want feedback

Reply:
Ah ok! that makes sense.